landing-page/content/common/puffin.md

```diff
@@ -0,0 +1,144 @@
+---
+url: puffin
```

Comment:
Can we rename this to `puffin-spec` to be clear in the URL?

Reply:
sure!